main.cc

```diff
@@ -1059,6 +1059,8 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             db.local().init_commitlog().get();
             db.invoke_on_all(&replica::database::start).get();
 
+            db.local().before_schema_keyspace_init();
+
```

Comment:
This should not be moved before `db::system_keyspace::enable_features_on_startup(feature_service).get();`, because before_schema_keyspace_init() examines features.

Reply:
Yes, I already figured that out, since my new test doesn't pass.